traffic_updater/pkg/trafficproxyclient/get_traffic_by_grpc_test.go

```diff
@@ -89,7 +100,7 @@ func TestGetDeltaTrafficDataByGRPCStreaming(t *testing.T) {
 	trafficDataChan := make(chan proxy.TrafficResponse)
 
 	go func() {
-		err := getDeltaTrafficFlowsIncidentsByGRPCStreaming(flags.trafficProxyFlags, trafficDataChan)
+		err := NewStreamingDeltaFlowsIncidents(trafficDataChan)
```

Comment:
Compare with GetFlowsIncidents, can we call this one GetStreamingDeltaFlowsIncidents
BTW, is it possible to put streaming controller in this function's parameter?  Like duration, etc.

Reply:
Yes, we can. But I still think the `get` prefix is not good enough for this function. It's used for set up a new channel for streaming, not just `get`. 
Agree that we still need to think about parameters of this function. 
Let me just remove the expose of this function first since it's not been actually used so far, then refactor it later when we need it in real application.